tools/frb_internal/lib/src/makefile_dart/consts.dart

```diff
@@ -36,9 +36,9 @@ const kDartExampleIntegratePackages = [
 
 const kDartExamplePackages = [
   'frb_example/dart_minimal',
+  'frb_example/dart_build_rs',
```

Comment:
nit: let's keep this unchanged

![image](https://github.com/user-attachments/assets/d4cc2652-8397-434f-86cf-7117535c2a44)